sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -222,6 +223,11 @@ def block_until_done(
         BigQueryJobCancelled exception to signify when that the job has been cancelled (i.e. from timeout or KeyboardInterrupt).
     """
 
+    # For test environments, retry more aggressively
```

Comment:
From a completely idealistic perspective, why couldn't we keep tests and the product in sync? What is the downside to just making all retries more aggressive, especially if we are just polling the status.

Reply:
Not much of one I guess? There's a cost of raise exceptions, but we're blocked anyways until the retry is done. We also don't really have other tasks going on.

I reduced the prod one to 1s + the test one to 0.1. In the test data, we know that our entity df is usually just 1 or 2 entities, whereas we expect much larger dataframes in prod use, so _seems_ like we should have something different.